CONTRIBUTING.md

```diff
@@ -60,6 +110,8 @@ There are several types of issues in the issue tracker:
 Issues can and should be added at anytime during development.
 If you see something, say something.
 
+See also [Issue Tracker](issue-tracker.md).
```

Comment:
Doesn't link, see previous comment above.

Reply:
This is real. Fixing it and the others.

Reply:
Thanks for catching it!